hatServer/static/js/services/surveyQuestions.js

```diff
@@ -0,0 +1,461 @@
+        weight: 1
+    };
+    var skillCategories = {
+        expert: true,
+        good: true,
+        basic: true,
+        none: true
+    };
+    var overallProgramming = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var databaseDevelopment = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var embeddedSystems = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var webApplications = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var mobileApplications = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var userInterface = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var statistics = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var networking = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var security = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var robotics = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var computerVision = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var algorithms = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var machineLearning = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var firstChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var firstComment = {
+        included: true,
+        required: false
+    };
+    var secondChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var secondComment = {
+        included: true,
+        required: false
+    };
+    var thirdChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var thirdComment = {
+        included: true,
+        required: false
+    };
+    var fourthChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var fourthComment = {
+        included: true,
+        required: false
+    };
+    var fifthChoice = {
+        included: true,
+        required: true,
+        weight: 2
+    };
+    var fifthComment = {
+        included: true,
+        required: false
+    };
+    var ipPreference = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var ipOptions = {
+        retain: 'Prefer to retain IP rights to my work',
+        retainIncluded: true,
+        depends: 'Depends on the project',
+        dependsIncluded: true,
+        noPref: 'I don\'t have a preference to retain IP rights to my work',
+        noPrefIncluded: true
+    };
+    var leadership = {
+        included: true,
+        required: true,
+        weight: 1
+    };
+    var leadershipOptions = {
```

Comment:
Same kind of deal here, STRONG_FOLLOW, PREFER_FOLLOW, NO_PREF, PREFER_LEAD, STRONG_LEAD. These were going to be enums, hence the caps...